src/arrayMethodSort.js

```diff
@@ -4,8 +4,21 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = function(a, b) {
+    return a.toString() > b.toString();
+  }) {
```

Comment:
For better readability default function should be like this
```suggestion
  function defaultCompareFunction (a, b) {
    return a.toString() > b.toString();
  }

  [].__proto__.sort2 = function(compareFunction = defaultCompareFunction) {
```